examples/radial_histogram_equalization.py

```diff
@@ -0,0 +1,108 @@
+"""
+=============================
+Radial Histogram Equalization
+=============================
+
+This example applies the Radial Histogram Equalizing Filter (`sunkit_image.radial.rhef`) to a sunpy map.
+"""
+
+import astropy.units as u
+import matplotlib.pyplot as plt
+import sunpy.data.sample
+import sunpy.map
+
+import sunkit_image.enhance as enhance
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+#######################################################################################
+# Let us use the sunpy sample data AIA image to showcase the RHE filter.
+
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+# Create radial segments (RHEF should use a dense grid)
+radial_bin_edges = equally_spaced_bins(0, 2, aia_map.data.shape[0] // 2)
+radial_bin_edges *= u.R_sun
+
+rhef_map = radial.rhef(aia_map, radial_bin_edges)
+
+fig, axes = plt.subplots(1, 2, figsize=(10, 5), sharex="all", sharey="all", subplot_kw={"projection": aia_map})
+
+aia_map.plot(axes=axes[0], clip_interval=(1, 99.99) * u.percent)
+axes[0].set_title("Original AIA Map")
+
+# By default, the new output map has the same normalization as the input map.
+# This means that when you plot it by default, the image is pretty pale.
+# So by setting it to None here, we bypass this issue.
+rhef_map.plot(axes=axes[1], norm=None)
```

Comment:
```suggestion
rhef_map.plot(axes=axes[1])
```
Since we are now setting the norm to be None.